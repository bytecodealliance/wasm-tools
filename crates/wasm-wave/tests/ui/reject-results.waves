result-ok-u8(ok);
result-ok-u8(ok());
result-ok-u8(o(0));
result-ok-u8(err(0));
result-err-u8(err);
result-err-u8(err());
result-err-u8(e(0));
result-err-u8(ok(0));
result-no-payloads(ok());
result-no-payloads(o(0));
result-no-payloads(ok(0));
result-no-payloads(err());
result-no-payloads(e(0));
result-no-payloads(err(0));
result-both-u8(ok);
result-both-u8(ok());
result-both-u8(o(0));
result-both-u8(err);
result-both-u8(err());
result-both-u8(e(0));