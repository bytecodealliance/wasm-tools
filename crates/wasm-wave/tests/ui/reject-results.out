error converting Wasm value: missing payload for "ok" case at 13..15
unexpected token: ParenClose at 16..17
invalid value type at 13..17
error converting Wasm value: unexpected payload for "err" case at 13..19
error converting Wasm value: missing payload for "err" case at 14..17
unexpected token: ParenClose at 18..19
invalid value type at 14..18
error converting Wasm value: unexpected payload for "ok" case at 14..19
unexpected token: ParenClose at 22..23
invalid value type at 19..23
error converting Wasm value: unexpected payload for "ok" case at 19..24
unexpected token: ParenClose at 23..24
invalid value type at 19..23
error converting Wasm value: unexpected payload for "err" case at 19..25
error converting Wasm value: missing payload for "ok" case at 15..17
unexpected token: ParenClose at 18..19
invalid value type at 15..19
error converting Wasm value: missing payload for "err" case at 15..18
unexpected token: ParenClose at 19..20
invalid value type at 15..19
